modules/peers.js

```diff
@@ -293,17 +293,18 @@ __private.dbSave = function (cb) {
 	}
 
 	// Wrap sql queries in transaction and execute
-	library.db.tx('modules:peers:dbSave', function (t) {
-		return t.peers.clear().then(function (value) {
-			return t.peers.insert(peers);
+	library.db.tx('modules:peers:dbSave', function * (t) {
+		yield t.peers.clear();
+		yield t.peers.insert(peers);
+	})
+		.then(() => {
+			library.logger.info('Peers exported to database');
+			return setImmediate(cb);
+		})
+		.catch(err => {
+			library.logger.error('Export peers to database failed', {error: err.message || err});
+			return setImmediate(cb);
```

Comment:
@vitaly-t For now we are only using ES6 inside DB repositories. I believe we should avoid mixing ES6 code in modules for now. Yes we will migrate all code to ES6 all together at some stage.

Reply:
@karmacoma What your intake on it?

Reply:
Once we got green light on ES6 within branch 1.0.0, no point holding back. And besides, once I refactor a task or transaction, moving it into ES6 land is the only proper solution.